Animations/Fade/UpBig.js

```diff
@@ -26,5 +28,5 @@ var FadeUpBig = {
     'transform': 'translateY(2000px)'
   }
 };
-//Use makeTransition function from main Tux Animation module to create wrapping animation componenet and pass in the default params
-module.exports = Animation.makeTransition('FadeUpBig', FadeUpBig);
+//Use makeAnimation function from main Tux Animation module to create wrapping animation componenet and pass in the default params
+module.exports = makeAnimation('FadeUpBig', FadeUpBig);
```

Comment:
This does not need to have `'FadeUpBig'` passed into it since we have defined a `className` property in `FadeUpBig`.